pkg/kubelet/cm/helpers_unsupported.go

```diff
@@ -41,7 +41,7 @@ func MilliCPUToQuota(milliCPU, period int64) int64 {
 }
 
 // MilliCPUToShares converts the milliCPU to CFS shares.
-func MilliCPUToShares(milliCPU int64) int64 {
+func MilliCPUToShares(milliCPU int64) uint64 {
```

Comment:
This is being fixed separately in #127293. After it is merged, I will rebase this PR.